### ISSUE TITLE
RFC for ImageQA Plugin

### ISSUE THREAD

**Issue by aray-wellcome** [wellcomecollection/goobi-infrastructure#293]
**The issue**
Our colleague, Rada, does sensitivity checking on each archival item we digitize. To sensitivity check, she needs to view every single image in an item, have full-screen views of the image, and be able to zoom into the image easily.

Prior to the migration she was downloading the images via the home directory folder and viewing the items in Adobe Bridge. With the migration to the Cloud, this is no longer a possibility. 

Her chief complaints about using Edit METS to view the images are:
1. The image is too small
2. The images (jpgs) are too slow to load, taking 1-3 seconds to load depending

**Possible resolution**
To resolve the images being too small, we could add the [ImageQA plugin](https://docs.goobi.io/goobi-workflow-plugins-en/step-plugins/intranda_step_imageqa) to the existing JP2_manual_edit_METS workflow (or a new one the replicates that workflow) before the Edit METS step. 

For the jpegs being too slow to load, we could test them on the ImageQA plugin to see if they are faster to switch between and if not, consider other solutions to this.

**Questions**
1. Is it possible to get a test version of the ImageQA plugin on staging Goobi to test if this is a good solution for Rada?
2. How much dev work will be needed to make the ImageQA plugin work in the Cloud?

**Comment by opaetzel** [comment]
We sped up the image loading in imageQA. This needs to be deployed to production.

**Comment by aray-wellcome** [comment]
Having talked to Rada, I have clarified her needs. 

For items she knows could be sensitive and they need a longer look or for items that are handwritten, she needs the full screen view. The full screen view size and speed is good for her but is it possible to disable the fade in when the jpegs load? It is hard to look at when going at speed. 

For items that are at low risk for being sensitive, she still needs to look over them but at speed. For this she would prefer having larger thumbnails. The thumbnails in the plugin is just too small still for a quick go-over. 

She is willing to sacrifice the size of the preview pane on the right-hand side of the screen for larger thumbnails as she can always use the full screen mode. 

When I talked to her, she said something perhaps this size might work:

![image](https://user-images.githubusercontent.com/49983203/82301467-36266080-99b0-11ea-80db-d48d2e5bf5c3.png)
